src/features/usage-consolidation-accordion/usage-consolidation-content-package.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  useIntl,
+  FormattedMessage,
+} from 'react-intl';
+import getSymbolFromCurrency from 'currency-symbol-map';
+
+import {
+  MultiColumnList,
+  Button,
+  Dropdown,
+  DropdownButton,
+  DropdownMenu,
+} from '@folio/stripes/components';
+
+import { costPerUse as costPerUseShape } from '../../constants';
+
+const propTypes = {
+  costPerUseData: costPerUseShape.CostPerUseReduxStateShape.isRequired,
+  year: PropTypes.string.isRequired,
+};
+
+const UsageConsolidationContentPackage = ({
+  costPerUseData,
+  year,
+}) => {
+  const intl = useIntl();
+  const {
+    isLoaded,
+    isFailed,
+    data,
+  } = costPerUseData;
+
+  if (!isLoaded && !isFailed) {
+    return null;
+  }
+
+  if (isFailed) {
+    return (
+      <div data-test-usage-consolidation-error>
+        <FormattedMessage id="ui-eholdings.usageConsolidation.summary.error" />
+      </div>
+    );
+  }
+
+  const currency = data?.attributes?.parameters?.currency;
+  const currencySymbol = getSymbolFromCurrency(currency);
+
+  const {
+    cost,
+    costPerUse,
+    usage,
+  } = data?.attributes?.analysis;
+  const noCostPerUseAvailable = !cost && !costPerUse && !usage;
+
+  const formatCost = (value) => `${currencySymbol || ''}${value} (${currency})`;
+  const formatValue = (value, formatter) => {
+    if (!noCostPerUseAvailable && !value) {
+      return '-';
+    }
+
+    return formatter ? formatter(value) : value;
+  };
+
+  if (noCostPerUseAvailable) {
+    return (
+      <div data-test-usage-consolidation-error>
+        <FormattedMessage
+          id="ui-eholdings.usageConsolidation.summary.noData"
+          values={{
+            record: 'package',
+            year,
+          }}
+        />
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <MultiColumnList
+        id="packageUsageConsolidationSummary"
+        onRowClick={() => {}}
+        contentData={[{ cost, costPerUse, usage }]}
+        visibleColumns={['cost', 'usage', 'costPerUse', 'ucActions']}
+        formatter={{
+          cost: (rowData) => formatValue(rowData.cost, formatCost),
+          costPerUse: (rowData) => formatValue(rowData.costPerUse, formatCost),
+          usage: (rowData) => formatValue(rowData.usage),
```

Comment:
Why do we need `formatValue` here? Without formatter this function returns value or dash so we can just write
```suggestion
          usage: (rowData) => rowData.usage || <NoValue />,
```
And if we do it `formatValue` function could loose condition in return

Reply:
I've missed a bug here, `formatValue` should check that the value is falsy, but not equal to 0 (added a test case for this).
If we check `!value && value !== 0` in `formatValue` then we can't use `||` to decide what to return here

Reply:
So, I clearly understand that we could use just `rowData.usage` here or it could be expect cases where we need `formatValue` function?

Reply:
For usage we also need to show either `0` or `-` depending on what value we get from back-end. And `formatValue` implement logic that decides what to show